app/services/submission/plant_population_finalizer.rb

```diff
@@ -30,7 +30,9 @@ def create_new_plant_lines
         taxonomy_term_id: taxonomy_term.id,
         entered_by_whom: submission.user.full_name,
         date_entered: Date.today,
-        user: submission.user
+        user: submission.user,
+        published: true,
+        published_on: Date.today
```

Comment:
Here you use Date.today yet in other places you set that column to Time.now. Which is correct?